### PR TITLE
Fix Persian install fixtures (missing 2 carrier entries)

### LINKED ISSUE #22657: Installation Error in Persian (Farsi) Language.
#### Describe the bug

On the installation process (PrestaShop v1.7.7.0) in Persian (Farsi) language, one error happened about carrier->delay is empty and installation with demo data can't be succeeded

I created a PR for this #22564

#### Expected behavior

Nobody can install PrestaShop in version 1.7.7.0.  
unless disabling demo data -OR- install in English and after that add the Persian Language from admin.
#### Steps to Reproduce

1. Go to the installation
2. change the language to Persian (Farsi)
3. Go to the installation steps.
4. See an error

**Screenshots**

image_2020-12-24_13-47-09

#### Additional information

* PrestaShop version: 1.7.7.0
* PHP version: I don't remember but I believe 7.1

### PR DESCRIPTION
**Pull request by AliShareei**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 1.7.7.x
| Description?      | IN: Fix installation - carrier delay empty (Error for Persian Language). On the installation process in Persian (Farsi) language, one error happened about carrier->delay is empty and installation with demo data can't be succeeded
| Type?             | bug fix
| Category?         | IN 
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | #22657
| How to test?      | just install PrestaShop with Persian (Farsi) language
| Possible impacts? | Installation process.

image_2020-12-24_13-47-09

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/22564)

### CODE REVIEW

**Comment by PierreRambaud** [comment]
PIng @LouiseBonnard it's up to you :D

**Comment by LouiseBonnard** [comment]
Hi @AliShareei, thanks a lot for your contribution! Can you please let us know which PrestaShop version(s) it is blocking?

**Comment by AliShareei** [comment]
Hi @LouiseBonnard, I saw this bug in the latest version of PrestaShop.
I mean 1.7.7.0 in the Persian language has this problem.

**Comment by LouiseBonnard** [comment]
Noted, thanks @AliShareei! Would you mind creating an issue reporting this bug and link your pull request to it? It'd be great as it'd allow us to reproduce and prioritize it. I would like to know if this is a regression (and therefore for which version it should be fixed) and if it also happens in the context of another RTL language.

**Comment by AliShareei** [comment]
Dear @LouiseBonnard, I have done it.
Actually, I don't know about the other RTL languages but I believe this problem is not related to Languages, It's about Demo data.

**Review by Progi1984 — DISMISSED** [review]

**Review by matks — DISMISSED** [review]

File: install-dev/fixtures/fashion/langs/fa/data/carrier.xml
Comment (Arman-Hosseini) [inline install-dev/fixtures/fashion/langs/fa/data/carrier.xml]:
```suggestion
    <delay>هرچه سبک تر ارزان تر!</delay>
```

**Review by Arman-Hosseini — DISMISSED** [review]
lgtm, thanks 👍🏻

**Review by Progi1984 — DISMISSED** [review]

**Comment by Arman-Hosseini** [comment]
@AliShareei, The PR should target `1.7.7.x` branch, As @matks said here: https://github.com/PrestaShop/PrestaShop/issues/22657#issuecomment-755386912

**Comment by AliShareei** [comment]
@Arman-Hosseini Jan merci babate peygiri! 
I changed the branch to 1.7.7.x.

**Comment by Arman-Hosseini** [comment]
> @Arman-Hosseini Jan merci babate peygiri!

Mamnunam Ali jan.

> I changed the branch to 1.7.7.x.

The base branch still targets the `develop` branch.
[This ref](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/changing-the-base-branch-of-a-pull-request) will help you.

**Comment by AliShareei** [comment]
@Arman-Hosseini Jan.

I think because I commit in develop the branch if I change the branch base to 1.7.7.x it's will import all commits from my fork develop branch to the 1.7.7.x branch.

am I wrong? Is there another way to solve the problem or can you solve it yourself?

change-branch

**Comment by Arman-Hosseini** [comment]
> @Arman-Hosseini Jan.
> 
> I think because I commit in develop the branch if I change the branch base to 1.7.7.x it's will import all commits from my fork develop branch to the 1.7.7.x branch.


You need to rebase.
If you can't, please request it from the maintainers.
Good luck!

**Comment by AliShareei** [comment]
@Arman-Hosseini Thank you.
I rebase it but it didn't work.
Maintainers Helpppppppppppppppp, please fix it 😄

**Comment by matks** [comment]
Hi @AliShareei you need to create a new branch from `1.7.7.x` and cherry-pick your commits into it.

Or if you are OK with that I can do it for you :) but I cannot use your branch so I'll create a new branch then. It will still be your commits but it will be my PR .

The branch is ready https://github.com/matks/PrestaShop/tree/fix-177x-persian

**Comment by AliShareei** [comment]
Hi @matks,
Thanks if you do this. ❤️
Maybe next time I will make the PR more carefully.

**Comment by matks** [comment]
See https://github.com/PrestaShop/PrestaShop/pull/22735 😉

**Comment by Julievrz** [comment]
As @Arman-Hosseini is now a proofreader on the Persian project on Crowdin I will trust him on this translation and remove the waiting for wording label. Thank you!

**Review by Progi1984 — APPROVED** [review]

**Comment by matks** [comment]
PR needs rebase as SCSS linter fails 😢

**Comment by Progi1984** [comment]
Fixed in 1.7.7.x (#22735). Will be available in 1.7.7.2, and in develop (soon after the next merge). Good job @AliShareei & @Arman-Hosseini.

**Pull request closed without merging**
